src/Sylius/Bundle/CoreBundle/EventListener/UserCartRecalculationListener.php

```diff
@@ -56,10 +56,6 @@ public function recalculateCartWhileLogin(Event $event)
             return;
         }
 
-        if (!$cart instanceof OrderInterface) {
-            throw new UnexpectedTypeException($cart, OrderInterface::class);
-        }
```

Comment:
Just a shortcut, sorry. This change is not a part of my intentional work it is a result of changes between branches.